src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ConnectionString.cs

```diff
@@ -20,6 +20,8 @@ public class ConnectionString
         private Dictionary<string, string> _keyValuePairs;
         private string _connString;
         private StringBuilder _parseErrorSb;
+        private string DEFAULT_TENANTID = "microsoft.onmicrosoft.com";
```

Comment:
@shahabhijeet I did not mean it is PII :) I was asking if it is PII. For me it is more desirable to have there explicit domain name under default tenant ID because if I have several subscriptions and different tenants in each of them it would be hard to remember during runtime/debug which tenant was used for login.